### ISSUE TITLE
Why does the cross-entropy loss can replace the policy gradient in the loss calculation of actor?

### ISSUE THREAD

**Issue by caidwang** [zhengsr3/Reinforcement_Learning_Pointer_Networks_TSP_Pytorch#5]
I notice that the loss of Actor is calculated by the following sentence 
```
path_loss+=self.Softmax_Cross_Entrophy(output_weight,torch.LongTensor(test_roads[:,i].T).squeeze())
```
where the test_roads is generated by ptr_net itself at the top of every train loop.

It confused me that how the policy gradient

$$\nabla_{\theta}J(\theta |s) \approx \frac{1}{B} \sum_{i=1}^{B} [(L(\pi_i|s_i) - b(s_i))\nabla_{\theta}logp_{\theta}(\pi_i | s_i)]$$

can be replaced by cross entropy ?
